app-vcs/lazygit/spec

```diff
@@ -0,0 +1,5 @@
+VER=0.43.1
+REL=1
```

Comment:
note: bump REL for topic Revision Marking Guidelines
in order to update package in the testing repository.